apps/legacy-api/src/controllers/PoolPairController.ts

```diff
@@ -513,3 +647,13 @@
 
   return undefined
 }
+
+export function encodeBase64 (next: NextToken): string {
+  return Buffer.from(JSON.stringify(next), 'utf8').toString('base64url')
```

Comment:
`toString(base64url) was introduced in v14.18` ref: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V14.md#14.18.0 so we'll need to set a dependency on this version in `jellyfish/package.json` (don't ask me how I realised this)

Reply:
Lemme guess - you were running with < v14.18 locally? LOL
So you're suggesting we bump https://github.com/DeFiCh/jellyfish/blob/ee2016fd86e172a53fd5ef5a2393173ce8fd9807/package.json#L17 to `>= 14.18` ?

If you have `nvm` installed, IDEA ide should automagically pick the node runtime when running anything in the project

Reply:
We don't have a .npmrc which enforces the version of node, so marking this as resolved